src/ApiStatus.ts

```diff
@@ -53,7 +53,15 @@ export default class ApiStatus {
         const content = message.cleanContent;
         if (!this.isApiStatusCommand(content)) return;
 
-        const apiStatus = await this.getApiStatus();
+        let apiStatus: StatusEmbedState;
+        try {
+            apiStatus = await this.getApiStatus();
+        } catch (error) {
+            console.error(error);
+            // fake status so we dont need to wrap the entire block
+            apiStatus = { api: {}, onFire: true, allApisOK: false, allApisIssues: true };
+            console.error(`Error occurred while making a request to the apistatus api: ${error}`);

```

Comment:
Printing the error twice 🤔

Reply:
Also, this will never get called since there's no way `getApiStatus` will reject (see comments below).